blockchain/msgs_test.go

```diff
@@ -97,7 +97,7 @@ func TestBlockchainMessageVectors(t *testing.T) {
 			BlockRequest: &bcproto.BlockRequest{Height: math.MaxInt64}}},
 			"0a0a08ffffffffffffffff7f"},
 		{"BlockResponseMessage", &bcproto.Message{Sum: &bcproto.Message_BlockResponse{
-			BlockResponse: &bcproto.BlockResponse{Block: bpb}}}, "1ac2020abf020a5d0a02080b1803220b088092b8c398feffffff012a021200380142204c149a7cfadc92b669b0cbfa4951a1b18c2d9f3177a3b8756d39ebb96e9d63317220e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b85512130a0b48656c6c6f20576f726c6412001a0022001ac8010a3000000000000000010000000000000001b81cb5596c28d044214b9f935e4af7dbe76e417f6182d86fbee68bfff7b2ff3a0a30ffffffffffffffffffffffffffffffffc4096ba8fccf882c309896e9168fa43fe62fccb752cb12d5160cc1d9c2ebffe7123000000000000000010000000000000001b81cb5596c28d044214b9f935e4af7dbe76e417f6182d86fbee68bfff7b2ff3a1230ffffffffffffffffffffffffffffffffc4096ba8fccf882c309896e9168fa43fe62fccb752cb12d5160cc1d9c2ebffe7"},
+			BlockResponse: &bcproto.BlockResponse{Block: bpb}}}, "1ac2020abf020a5d0a02080b1803220b088092b8c398feffffff012a02120038014220269ece38583f42aaf53fdd3abe1f570ab9b0d08284d080900966040a29df504c7220e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b85512130a0b48656c6c6f20576f726c6412001a0022001ac8010a3000000000000000010000000000000001163ff985bddcd0016e2535fc0e156f482c7df36c7a0c0cabc026d9d45d7a81d60a30ffffffffffffffffffffffffffffffffaf204cfc430a1f342d321b95c49392843e165ee721c5cf151bed7f5bab67f408123000000000000000010000000000000001163ff985bddcd0016e2535fc0e156f482c7df36c7a0c0cabc026d9d45d7a81d61230ffffffffffffffffffffffffffffffffaf204cfc430a1f342d321b95c49392843e165ee721c5cf151bed7f5bab67f408"},
```

Comment:
I just updated this PR to the latest version of nmt, which actually changes the way we hash things, so this is expected